### PR TITLE
DEPL-9892 - Set up development infrastructure for Oracle SOA Suite plugin

### PR DESCRIPTION
**Pull request by xkapil**
Configuring `master` branch to use Vanilla Gradle

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,24 +1,59 @@
+buildscript {
+  repositories {
+    mavenLocal()
```
Comment (hierynomus) [inline build.gradle]: Is mavenLocal needed?
Is mavenCentral needed?

(i.e. nexus mirrors mavenCentral, and mavenLocal is only needed when you're using unreleased plugins), so I think those two should go...

File: build.gradle [hunk 2]
```diff
@@ -1,24 +1,59 @@
...
+        credentials {
+          username nexusUserName
+          password nexusPassword
+        }
+      }
+    }
+  }
+
+  dependencies {
+    classpath "com.xebialabs.gradle.plugins:gradle-xl-plugins-plugin:${xlPluginPluginVersion}"
```
Comment (hierynomus) [inline build.gradle]: The correct key would be `xlPluginsPluginVersion` (mind the extra 's')

File: build.gradle [hunk 3]
```diff
@@ -1,24 +1,59 @@
...
+  }
+}
+
 plugins {
     id "com.github.hierynomus.license" version "0.11.0"
-    id "com.xebialabs.xldp" version "1.0.5"
+//    id "com.xebialabs.xldp" version "1.0.5"
```
Comment (hierynomus) [inline build.gradle]: Our code is not an museum ;)

File: build.gradle [hunk 4]
```diff
@@ -1,24 +1,59 @@
...
 sourceCompatibility = 1.7
 targetCompatibility = 1.7
-version = '1.5.0'
+//version = '1.5.0'
 
-repositories {
-    mavenLocal()
-    mavenCentral()
-}
+//repositories {
```
Comment (hierynomus) [inline build.gradle]: Museum code

File: build.gradle [hunk 5]
```diff
@@ -28,13 +63,13 @@ dependencies {
 sourceSets.main.resources.srcDirs = ["src/main/jython", "src/main/resources"]
 sourceSets.test.resources.srcDirs = ["src/test/jython", "src/test/resources"]
 
-task testJython(type: Exec, dependsOn: ['testClasses']) {
+/*task testJython(type: Exec, dependsOn: ['testClasses']) {
```
Comment (hierynomus) [inline build.gradle]: Does this task come automatically from one of our plugins? Why is it commented out?

File: gradle/dependencies.conf [hunk 6]
```diff
@@ -0,0 +1,8 @@
+dependencyManagement {
+  versions {
+    docBaseStyleVersion="2.0.11"
```
Comment (hierynomus) [inline gradle/dependencies.conf]: How many of these keys come from xld-reference already? I think a few at least

**Comment by hierynomus** [comment]
Some of the comments also hold for other already migrated plugins, might be good to make another pass over them if you have some time :)

File: gradle/dependencies.conf (hunk 6 above)
Comment (xkapil) [inline gradle/dependencies.conf]: We couldn't identify any predefined dependency. They are being referred by key in xl/xld-reference. While making these changes we had referred to Ric's PR - https://github.com/xebialabs/tomcat-plugin/pull/15

File: build.gradle (hunk 5 above)
Comment (xkapil) [inline build.gradle]: The comment should go away. It doesn't comes from the plugin. Currently, we are trying to remove the system's dependency of having jythonInterpreter to build.

**Pull request merged**